### ISSUE TITLE
Cocaine is deprecated

### ISSUE THREAD

**Issue by jkubacki** [musaffa/file_validators#24]
DEPRECATION: The cocaine gem is deprecated. Please upgrade to terrapin. See https://github.com/thoughtbot/terrapin/ for further instructions.

**Comment by musaffa** [comment]
Thanks for notifying. Before migrating to terrapin, I will see whether cocaine/terrapin dependency can be removed from this gem.

**Comment by one-more-alex** [comment]
Please see my proposition to get rid of cocaine/terrapin dependency in PR https://github.com/musaffa/file_validators/pull/25

**Comment by toasterlovin** [comment]
@musaffa Would you be open to a release which migrates to Terrapin before possibly removing both as a dependency? I have a project where it would be nice to remove the deprecation warning. If it's really as easy as Terrapin makes it out (basically, change the gem dependency and change constant names), then I would be happy to send a PR...

**Comment by musaffa** [comment]
@toasterlovin Yes you can send a PR with Terrapin.

**Comment by Conorbro** [comment]
Thanks for merging 👍 

Can we expect a new release any time soon? 😃

**Comment by Conorbro** [comment]
@musaffa Can we expect a new release any time soon? 😃

**Comment by musaffa** [comment]
Yes. Will release a beta. This is going to be a major version bump.

**Comment by toasterlovin** [comment]
I finally got some time to get to this and it turns out that @Conorbro already has a PR merged. Thank you. 🙏🏻
